ts/Core/Axis/ColorAxis.ts

```diff
@@ -1292,6 +1292,14 @@ class ColorAxis extends Axis implements AxisLike {
         chart.isDirtyLegend = true;
     }
 
+    // Removing the whole axis (#14283)
+    public destroy(): void {
+        this.chart.isDirtyLegend = true;
+
+        this.destroyItems();
+        H.Axis.prototype.destroy.call(this);
```

Comment:
Use `super.destroy();`. In addition, we probably need to pass over the arguments (`keepEvents`, or more future proof, all arguments) to the super method.